schedule_handler.js

```diff
@@ -1,11 +1,12 @@
-var AWS = require('aws-sdk');
+import { ECSClient, RunTaskCommand } from "@aws-sdk/client-ecs";
+
 var util = require('util');
 
 exports.handler = function(input, context) {
```

Comment:
```suggestion
exports.handler = async function(input, context) {
```

Since there's an `await` down below, we'll need this to be `async`.

Reply:
Nice catch. Thanks